src/Interpreter.cpp

```diff
@@ -38,10 +39,10 @@ void Interpreter::processCommand(const string &command) {
         return;
     }
 
-    if (tokens.size() < 2) {
-        cout << "" << endl;
-        return;
-    }
+    // if (tokens.size() < 2) {
+    //     cout << "" << endl;
+    //     return;
+    // }
```

Comment:
couldn't get the `help` command to run below since we always early returned here. I think it should be fine to just remove this, but wanted to check in on @suranimaria first.

also, I think we might need one more command - something like `view` to see all the tables we have.